scripts/tools-versions-linux.txt

```diff
@@ -1,13 +1,12 @@
-python3=3.8.0-3~18.04
-git=1:2.17.1-1ubuntu0.7
-cmake=3.17.3-0kitware1
-ninja-build=1.8.2-1
-gperf=3.1-1
-dtc=1.4.7-3ubuntu2
+python3=3.8.2
+git=1:2.25.1
+cmake=3.17.3
+ninja-build=1.9.0
+gperf=3.1
+dtc=1.4.7
 gnuarmemb=9-2019-q4-major
 gnuarmemb-short=9-2019q4
-ses=5.30a
-ccache=3.4.1-1
+ses=5.34a
+ccache=3.7.7
 dfu_util=0.9-1
-west=0.8.0
-gn=1880
```

Comment:
Why is this line being deleted ?
This causes the doc build to fail.

Reply:
thx, didn't mean to.  was a conflict resolution mistake.